client/public/assets/draco/draco_decoder.js

```diff
@@ -0,0 +1,52 @@
+
+var DracoDecoderModule = (function() {
```

Comment:
Why is this draco in public assets and not in a node module?

Reply:
i will remove this 
it is an error that it was commited